testgrid/config/config.yaml

```diff
@@ -1039,10 +1039,12 @@ dashboards:
   - name: gce-ha-master
     test_group_name: kubernetes-e2e-gce-ha-master
 
-- name: google-gci
+# "google-gci-dev" is a tab for tests that are used for GCI(aka COS) image
+# development and qualification. It should not be confused with the "google-gci"
+# tab that is for k8s development and release qualification.
+# If you are going to make a change here, kindly cc @kubernetes/goog-image.
+- name: google-gci-dev
```

Comment:
The idea behind suggesting 'google-gci-qa' name was to indicate the _purpose_ of the tests, rather than what tests it contains or what they are named. But this is a moot point.